library/components/TNavigationDropdown.vue

```diff
@@ -43,7 +43,7 @@
 					:key='index'
 					:class="navSelected(page.url, 'text-[#1c1c21]', 'text-[#555463]')"
 				>
-					<a :href="page.url" class="flex items-center justify-between w-full">
+					<button @click="openReport(page)" class="flex items-center justify-between w-full">
```

Comment:
If I understand correctly, you would like to know when a user opens another report, using this element?
Is that something we can collect using the page visit we already gather when a page is opened?

Reply:
Alternatively, if you can still fire the tracking event and also leave this as a link that would be grand. I am not familiar with triggering an onclick and also a navigation but there are some guidelines here: 

https://www.w3.org/WAI/WCAG21/Techniques/client-side-script/SCR35

I tried it out, but I honestly am not sure how this will behave with our iframe situation: https://codepen.io/sareg0/pen/BaxBEPN?editors=1001

Reply:
The problem is, this event is specially made for this dropdown and I doubt we can track it as anchor tag, because while we can detect @click, we cannot detect open in new tab.

And maybe I am wrong in this but these this dropdown is more like a selector that is switching reports (from an user's perspective) instead of navigation to a different page.

Reply:
Perhaps we need to introduce a new mixin. There may also be a future where we wrap all of our components, and this wrapper is responsible for tracking and what not. 🤷 It's hard to tell where we're going so going with this is fine for now.

Reply:
> And maybe I am wrong in this but these this dropdown is more like a selector that is switching reports (from an user's perspective) instead of navigation to a different page.

Still, anything that changes navigation is most suitable as a link.